modules/oe_theme_content_event/src/Plugin/ExtraField/Display/LivestreamExtraField.php

```diff
@@ -99,6 +99,17 @@ public function viewElements(ContentEntityInterface $entity) {
     // render the date only.
     if ($event->isOnlinePeriodYetToCome($this->requestDateTime)) {
       $this->applyHourTag($build, $event->getOnlineStartDate());
+      $current_date = $this->dateFormatter->format($this->requestDateTime->getTimestamp(), 'custom', 'Ymd');
+      $start_day = $this->dateFormatter->format($event->getOnlineStartDate()->getTimestamp(), 'custom', 'Ymd');
+      if ($current_date === $start_day) {
+        $build['#hide_link'] = TRUE;
+        $build['#attached'] = [
+          'library' => 'oe_theme_content_event/livestream_link_disclosure',
+          'drupalSettings' => [
```

Comment:
Good approach is to add settings to module's section in drupalSettings.
```suggestion
          'drupalSettings' => [
              'oe_theme_content_event' => [
                      'livestream_starttime_timestamp' => $event->getOnlineStartDate()->getTimestamp() * 1000,
                  ],
          ],
```
Then it can be used as drupalSettings.oe_theme_content_event.livestream_starttime_timestamp

Reply:
done